FirebaseMLModelDownloader/Sources/ModelDownloadTask.swift

```diff
@@ -163,3 +201,22 @@ extension ModelDownloadTask: URLSessionDownloadDelegate {
     }
   }
 }
+
+/// Extension to handle background downloading.
+extension ModelDownloadTask: UIApplicationDelegate {
+  /// Save completion handler for background download.
+  func application(_ application: UIApplication,
+                   handleEventsForBackgroundURLSession identifier: String,
+                   completionHandler: @escaping () -> Void) {
+    assert(identifier == backgroundSessionIdentifier)
```

Comment:
If the host app also uses the background downloading, then we will crash (at least in Debug) when handling events for the host app URL session.